indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -1,86 +1,145 @@
+                                                </a>
+                                            </li>
+                                            <li>
+                                                <a data-href="{{ url_for('.email_tpl_text_edit', tpl) }}"
+                                                   title="{% trans %}Edit e-mail text{% endtrans %}"
+                                                   data-title="{% trans %}Edit e-mail text{% endtrans %}"
+                                                   data-update="#email-template-manager"
+                                                   data-ajax-dialog>
+                                                   {% trans %}E-mail text{% endtrans %}
+                                                </a>
+                                            </li>
+                                        </ul>
+                                        <a href="#" class="i-button js-delete-item icon-remove icon-only"
+                                           title="{% trans %}Delete template{% endtrans %}"
+                                           data-qtip-style="danger"
+                                           data-href="{{ url_for('.email_tpl_delete', tpl) }}"
+                                           data-method="DELETE"
+                                           data-confirm="{% trans %}Are you sure you want to delete this template?{% endtrans %}"
+                                           data-title="{% trans %}Delete template{% endtrans %}"
+                                           data-update="#email-template-manager">
                                         </a>
-                                    </li>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="i-box-content">
+                                <div class="right">
+                                    <a class="email-preview-btn"
+                                       id="email-preview-btn-{{ tpl.id }}"
+                                       data-id="{{ tpl.id }}"
+                                       data-visible>
+                                        {%- trans %}Show preview{% endtrans -%}
+                                    </a>
+                                </div>
+                                <div>
+                                    {%- trans %}This email will be sent when the abstract is:{% endtrans -%}
+                                </div>
+                                <ul>
+                                    {% for rule in tpl.rules if rule %}
+                                        <li class="notification-rule">
+                                            <strong>{{ states[rule.state[0]] }}</strong>
+                                            {% if 'track' in rule %}
+                                                {%- trans %}in{% endtrans -%}
```

Comment:
For better i18n, I would repeat `in` within each one of the tralatable strings below. They would look like

```jinja
{%- trans -%}
    in <strong>no track</strong>
{%- endtrans -%}
```